lib/error-matcher.js

```diff
@@ -77,12 +90,22 @@ module.exports = (function () {
   };
 
   ErrorMatcher.prototype.set = function (regex, cwd, output) {
-    try {
-      this.regex = XRegExp(regex);
-    } catch (err) {
-      this.regex = null;
-      return this.emit('error', 'Error parsing regex:\n' + err);
+    this.regex = [];
+    for (var i in regex) {
+      var curRegex = null;
+      try {
+        curRegex = XRegExp(regex[i]);
+      } catch (err) {
+        this.emit('error', 'Error parsing regex:\n' + err);
+      }
+      if (curRegex !== null) {
+        this.regex.push(curRegex);
+      }
     }
+    if (this.regex.length === 0) {
+      this.regex = 0;
+    }
+
```

Comment:
Not sure why you would want to do this? Seems like just leaving it an empty array will also cause the loop on row `64` not to execute. Besides, `this.regex` will always be an array as well.